scanner/go.mod

```diff
@@ -5,22 +5,35 @@ go 1.20
 require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/pkg/errors v0.9.1
```

Comment:
https://github.com/pkg/errors is no longer maintained. Do we need it? Will the builtin `errors` package not be sufficient?

Reply:
Eh, we already have an indirect dep on `pkg/errors`, but it's cheap right now to move out: 7c3d8cee